iib/workers/tasks/fbc_utils.py

```diff
@@ -17,6 +19,45 @@ def is_image_fbc(image):
     :return: True if image is FBC type, False otherwise (SQLite)
     :rtype: bool
     """
+    from iib.workers.tasks.utils import skopeo_inspect
+
     skopeo_output = skopeo_inspect(f'docker://{image}')
     dc_image_label = 'operators.operatorframework.io.index.configs.v1'
```

Comment:
Minor and unrelated observation: dc_image_label should be renamed to fbc_image_label in a separate PR